test/integration/GuardedStream.test.ts

```diff
@@ -43,7 +43,7 @@ describe('A chained converter where data gets ignored', (): void => {
 
   it('does not throw on async crash.', async(): Promise<void> => {
     jest.useFakeTimers();
-    const result = await converter.handleSafe({ identifier, representation: rep, preferences: {}});
+    const result = await converter.handleSafe({ identifier, representation: rep, preferences: { type: { 'x/x': 1 }}});
```

Comment:
I'm a bit worried that the integration test had to change. Seems like the original should still work.

Reply:
This might actually be related to the issue I described of matching `internal/quads` to `*/*` because I think it's the same function that handles that.

Reply:
Going through this now, it actually still has to change since we were testing a chain of 2 converters here. With the new ChainedConverter it would be too smart and do the conversion with a path of length 1.